File: Programlar/C/50sual FM (Olimpiada)/A45.cpp
#include<stdio.h>
int main(){
	float a,b,c;
	scanf("%d%d%d",&a,&b,&c);
	if(a+b>c and a+c>b and b+c>a){
		if(a*a + b*b == c*c)printf("duzbucaqli");
		else if(a*a + b*b < c*c)printf("korbucaqli");
		else printf("itibucaqli");
	}
	else printf("bele ucbucag yoxdur");
	
return 0;
}

//terefleri daxil edilmis ucbucagin hansi novune aid oldugunu teyin edir

File: Programlar/Python/50sual FM (Olimpiada)/A45.py
a = int(input())
b = int(input())
c = int(input())
if a+b>c and b+c>a and a+c>b:
    if a*a + b*b == c*c: print("Duzbucaqli")
    elif a*a + b*b < c*c: print("Korbucaqli")
    else: print("Itibucaqli")
else: print("Bele ucbucag yoxdur!")

#terefleri daxil edilmis ucbucagin hansi novune aid oldugunu cap edir